suite-native/module-home/src/screens/HomeScreen/components/BiometricsBottomSheetIcons.tsx

```diff
@@ -0,0 +1,47 @@
+import { Platform } from 'react-native';
+
+import { IconName, Icon } from '@suite-common/icons';
+import { HStack, Box } from '@suite-native/atoms';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+const ICON_SIZE = 64;
+const ICON_WRAPPER_SIZE = 88;
+
+const iconWrapperStyle = prepareNativeStyle(utils => ({
+    padding: 12,
+    borderRadius: utils.borders.radii.round,
+    backgroundColor: utils.colors.backgroundSurfaceElevation2,
+    color: utils.colors.iconPrimaryDefault,
+    width: ICON_WRAPPER_SIZE,
+    height: ICON_WRAPPER_SIZE,
+}));
+
+export const BiometricsBottomSheetIcons = ({
+    isFacialEnabled,
+    isFingerprintEnabled,
+}: {
+    isFacialEnabled: boolean;
+    isFingerprintEnabled: boolean;
+}) => {
+    const { applyStyle } = useNativeStyles();
+
+    const faceRecognitionIcon: IconName = Platform.OS === 'ios' ? 'faceId' : 'face';
```

Comment:
what about to use [`isIOs()` / `isAndroid()` utilities](https://github.com/trezor/trezor-suite/blob/develop/packages/env-utils/src/envUtils.native.ts/#L59)? I personally would find it more readable.

Reply:
I believe `Platform.OS === 'ios'` is very common in the react native code in general and we use it like that at many places in our codebase. On the other hand we don't use `isIOS / isAndroid()` almost anywhere ATM.

If you strongly insist on using these two utility functions, let's discuss it whole team together. So we have consensus and all of us use it.

Reply:
It's definitely a nitpick, let's discuss in team what we will prefer 👌